notifications_utils/recipients.py

```diff
@@ -572,13 +572,15 @@ def validate_phone_number(number, international=False):
     number = normalise_phone_number(number)
 
     if len(number) < 8:
-        raise InvalidPhoneError("Not enough digits")
+        raise InvalidPhoneError("Mobile phone number must be at least 8 digits long")
 
     if len(number) > 15:
-        raise InvalidPhoneError("Too many digits")
+        raise InvalidPhoneError("Mobile phone number cannot be longer than 15 digits")
 
     if get_international_prefix(number) is None:
-        raise InvalidPhoneError("Not a valid country prefix")
+        raise InvalidPhoneError(
+            "The country code is not correct - double check the first 2 digits of this mobile number"
```

Comment:
Just in case you aren't aware, not all country codes are 2 digits long. See https://en.wikipedia.org/wiki/List_of_country_calling_codes

Some are 3 and some are even 4 digits long...

Nice and confusing...

Reply:
```suggestion
            "Country code not found - double check this mobile number"
```